web/html/src/components/dialog/ModalButton.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { Button } from "../buttons";
+import { showDialog } from "./util";
+
+// TODO: Extend Button's props here once that branch gets merged
```

Comment:
Make sure to not forget about this, we try to avoid merging `TODO`s , but I also see a reason here so, just keep it in mind to get rid of it ASAP, or track it into a proper issue at least.

Reply:
That's a fair point, a number of places in the migration process rely on essentially "fix this once something else gets done as well" which may be weeks apart. I'll add a ticket to look over all of these, thanks for the suggestion. 👍

Reply:
Created https://github.com/SUSE/spacewalk/issues/13952